src/components/Card/Card.stories.tsx

```diff
@@ -17,299 +18,91 @@ export default {
-            <div>Written in English</div>
-            <div>Under Creative Commons License</div>
-        </>
-    </Card>
-);
+const imageRatios = {
+    "1x1": "https://placeimg.com/100/100/animals",
+    "2x1": "https://placeimg.com/200/100/animals",
+    "3x4": "https://placeimg.com/150/200/animals",
+    "4x1": "https://placeimg.com/400/100/animals",
+    "4x3": "https://placeimg.com/200/150/animals",
+    "16x9": "https://placeimg.com/400/225/animals",
+};
 
-export const editionCardWithSomeEditionInfo = () => (
-    <Card
-        id="card#1"
-        className="edition-card"
-        image={
-            <Image
-                src="https://placeimg.com/300/400/arch"
-                isDecorative={true}
-            />
-        }
-        heading={
-            <Heading
-                level={2}
-                id="editioncardheading1"
-                url="#edition-link"
-                text={"2004 Edition"}
-            />
-        }
-        ctas={
-            <div className="edition-card__ctas">
-                <Link linkType={LinkTypes.Button} href="blah">
-                    Read Online
-                </Link>
-                <div className="edition-card__download">
-                    <Link href="#url" linkType={LinkTypes.Action}>
-                        <Icon
-                            name="download"
-                            blockName="more-link"
-                            decorative={true}
-                            modifiers={["left"]}
-                            iconRotation={IconRotationTypes.rotate0}
-                        ></Icon>
-                        Download
-                    </Link>
-                </div>
-            </div>
-        }
-    >
-        <>
-            <div>Published in New York</div>
-            <div>Under Creative Commons License</div>
-        </>
-    </Card>
+export const card = () => (
+    <>
+        {boolean("Show Header", true)
+            ? (showHeader = true)
+            : (showHeader = false)}
+        {boolean("Show Image", true) ? (showImage = true) : (showImage = false)}
+        {boolean("Show CTAs", true) ? (showCTAs = true) : (showCTAs = false)}
+        {boolean("Show Footer", true)
+            ? (showFooter = true)
+            : (showFooter = false)}
+        <Card
+            id="cardID"
+            heading={
+                showHeader ? (
+                    <Heading
+                        level={select(
+                            "Header Level",
+                            headerLevels,
+                            headerLevels[2]
+                        )}
+                        id="heading1"
+                        text={text("Header Content", "Optional Header")}
+                    />
+                ) : (
+                    false
+                )
+            }
+            image={
+                showImage ? (
+                    <Image
+                        src={select("Image Ratio", imageRatios, imageRatios[2])}
```

Comment:
Changing the image ratio doesn't seem to change the size of the container, just the content - is that intended?  Is this a change to the card from when it was designed in DRB?)

Reply:
Yeah, that was intended. DRB should have always had this functionality imo, I think the fact that it didn't was an oversight

Reply:
This is a thing we're gonna have to communicate to Sean, otherwise it'll be considered a bug ... I think we're pretty used to seeing it the other way.  The Metronome that I built DRB off of had the border sticking to the book, I think.

Reply:
This is also the kind of thing that I think would be nice to put in a PM-facing changelog.

Reply:
[Here's a link to those designs](https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=17167%3A58131). I don't see a border around the book, but it is a change. Should definitely be called out since it's a change we want QA to be aware of, maybe that's what we surface with PMs?

Reply:
I've updated the CHANGELOG with the change as well, since that should have been in there separately from the note about `.edition-card` styles going away.